subnet/kube/kube.go

```diff
@@ -403,6 +403,7 @@ func (ksm *kubeSubnetManager) nodeToLease(n v1.Node) (l subnet.Lease, err error)
 		l.Attrs.BackendV6Data = json.RawMessage(n.Annotations[ksm.annotations.BackendV6Data])
 
 		ipv6Cidr := new(net.IPNet)
+		log.Infof("This is the n.Spec.PodCIDRs: %v", n.Spec.PodCIDRs)
```

Comment:
Great idea! Can you add something more descriptive of where we are in the code? For example: "About to create a node lease for IPv6. This is n.Spec.PodCIDRs: %v"?